experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -174,65 +132,73 @@ export interface MoveOut<TNodeChange = NodeChangeType>
 	extends HasRevisionTag,
 		HasMoveId,
 		HasChanges<TNodeChange>,
-		CanConflict {
+		CellTargetingMark {
 	type: "MoveOut";
 	count: NodeCount;
-	/**
-	 * When true, the corresponding MoveIn has a conflict.
-	 * This is independent of whether this mark has a conflict.
-	 */
-	isDstConflicted?: true;
 }
 
 export interface HasReattachFields extends HasPlaceFields {
 	/**
-	 * The tag of the change that detached the data being reattached.
-	 *
-	 * Undefined when the reattach is the product of a tag-less change being inverted.
-	 * It is invalid to try convert such a reattach mark to a delta.
+	 * The revision this mark is inverting a detach from.
+	 * If defined this mark is a revert-only inverse,
+	 * meaning that it will only reattach nodes if those nodes were last detached by `inverseOf`.
+	 * If `inverseOf` is undefined, this mark will reattach nodes regardless of when they were last detached.
 	 */
-	detachedBy: RevisionTag | undefined;
+	inverseOf?: RevisionTag;
+}
 
-	/**
-	 * The original field index of the detached node(s).
-	 * "Original" here means before the change that detached them was applied.
-	 */
-	detachIndex: number;
+export interface DetachEvent {
+	revision: RevisionTag;
```

Comment:
Consider adding a comment to explain that this is the intention of the edit that performed the detach. Otherwise just rename to "intention".